sdk/core/azure-core-tracing-opentelemetry/inc/azure/core/tracing.hpp

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+#pragma once
+
+#include "azure/core/tracing/opentelemetry/dll_import_export.hpp"
+#include "azure/core/tracing/opentelemetry/internal/apiview.hpp"
```

Comment:
`/internal/` shouldn't be included.

Reply:
DOne. This file was created before I added the "remove internal" filter.